config.w32

```diff
@@ -176,7 +191,14 @@ if (PHP_MONGODB != "no") {
     MONGOC_HAVE_RES_SEARCH: 0,
     MONGOC_CC: "",
     MONGOC_USER_SET_CFLAGS: "",
-    MONGOC_USER_SET_LDFLAGS: ""
+    MONGOC_USER_SET_LDFLAGS: "",
+    MONGOCRYPT_ENABLE_TRACE: 1,
```

Comment:
On second thought, does it make sense to _not_ define any `MONGOCRYPT_` vars in the initial `mongoc_opts` definition and instead defer to the `if (PHP_CLIENT_SIDE_ENCRYPTION != "no")` condition later on? These options are only relevant to replacing terms in the header input files when we call `mongodb_generate_header()`, but we'll only call that from within the conditional anyway. If so, I'd suggest creating a `mongocrypt_opts` object, so we don't add unnecessary fields into `mongoc_opts`. That should also test that the replacements in the libmongocrypt configs don't require anything beyond what we'll put in `mongocrypt_opts`.

`MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION` is the only definition that needs to stay with an initial definition of `0`, since we need that to generate the libmongoc configs either way.